rfc/README.md

```diff
@@ -0,0 +1,46 @@
+# The OpenTofu RFC process
+
+If a larger body of work is needed for an enhancement in the [main OpenTofu repository](https://github.com/opentofu/opentofu) or other repositories, the core team often asks for an RFC to be written. We do this in order to make sure everyone is on the same page how a feature should be implemented. This repository holds all past RFCs and anyone can contribute RFCs.
+
+## Filing an RFC
+
+Filing and getting an RFC merged is intentionally a longer process. We use pull requests to collaboratively work on a technical solution to a problem.
+
+Before filing an RFC, ideally open an enhancement issue in the [main repository](https://github.com/opentofu/opentofu) in order to discuss if the feature has enough community support to warrant working on an RFC. This step is optional, but highly recommended.
+
+To file an actual RFC, please create a PR by copying the [template](template/) into a new folder in the [accepted](accepted/) folder. Please name the new folder according to the following template:
```

Comment:
"an actual" -> "a"

Reply:
Do we need an accepted folder?  I guess this brings up the question of what happens if an RFC is rejected?

Reply:
Why would we need an accepted folder?  The draft would only exist within the PR and would not be available in the repo until it's merged?